ale_linters/rust/cargo.vim

```diff
@@ -45,6 +46,16 @@ function! ale_linters#rust#cargo#GetCommand(buffer, version_output) abort
         let l:include_features = ' --features ' . ale#Escape(l:include_features)
     endif
 
+    let l:avoid_whole_workspace = ale#Var(a:buffer, 'rust_cargo_avoid_whole_workspace')
+    let l:nearest_cargo_prefix = ''
+    if l:avoid_whole_workspace
+      let l:nearest_cargo = ale#path#FindNearestFile(a:buffer, 'Cargo.toml')
+      let l:nearest_cargo_dir = fnamemodify(l:nearest_cargo, ':h')
+      if l:nearest_cargo_dir isnot# '.'
```

Comment:
Use 4 space indentation in Vim files. Add a blank line before every `if` which doesn't start a new level of indentation.